python/sdk/test/client_test.py

```diff
@@ -353,6 +353,22 @@ def mock_oauth():
         "aliases": ["default"]
     }
     """, status=200, content_type='application/json')
+
+    responses.add('GET',
```

Comment:
Thanks for this change! Btw, the `mock_oauth` fixture is defined in 2 files (this one and `merlin_test.py`). Shall we make it a common fixture in a new `fixtures.py` or something?

Reply:
Refactored by merging mock_oauth in client_test.py and merlin_test.py to one in conftest.py. Thanks for the suggestion!